src/www/widgets/widgets/carp_status.widget.php

```diff
@@ -74,14 +74,14 @@
 }
 if ($ipaddress) {
 ?> &nbsp;
-        <?=htmlspecialchars($status);?> &nbsp;
+        <?=htmlspecialchars(gettext($status));?> &nbsp;
```

Comment:
It's not worth fixing for now. Typical chicken-and-egg problems. Until we have a good model that wraps all of this we are stuck here (strings vs. integers for state). :)